.github/workflows/release.yml

```diff
@@ -1,17 +1,23 @@
 name: "release"
 on:
   push:
-    tags:
-      - '*'
+    tags: # Will upload to luarocks.org
+      - "*"
+  pull_request: # Will test a local install without uploading to luarocks.org
```

Comment:
Is that really necessary?

Reply:
this allows to check the rockspec doesn't get broken. Some lua packages relies on `luarocks test` to run tests.
What I could do is here is run the action only when the github workflow or template is changed.

Reply:
Yes, that would be better. Let's start with template only.